src/services/mapService.tsx

```diff
@@ -57,7 +57,6 @@ export async function fetchRegionDatetimes(aoiId: AoiId): Promise<number[]> {
         console.error('Error loading region datetimes:', error)
         throw error
     }
-    //   todo use tanstack query
```

Comment:
Line 32:

function transformRegionDatetimes(request: IAPIRegionDatetimes): number[] {
    return Object.keys(request).map((timestamp) => parseInt(timestamp, 10))
    // TODO: fix this (include the imageData)
    // return Object.entries(request).map(([timestamp, imageData]) => ({
    //     timestamp: parseInt(timestamp, 10),
    //     // imageData: imageData,
    // }))
}

Is this fixed/ is it essential to fix it?